doc/modules/linear_model.rst

```diff
@@ -1291,7 +1344,7 @@ E.g., with ``loss="log"``, :class:`SGDClassifier`
 fits a logistic regression model,
 while with ``loss="hinge"`` it fits a linear support vector machine (SVM).
 
-.. topic:: References
+.. topic:: References:
```

Comment:
This is weird here because this is not really a reference as usual. This is a cross-link to another part of the documetation. I would therefore make a sentence instead of a specific section:

```
You can refer to the dedicated :ref:`sgd` documentation section for more details.
```